doma-spring-boot-samples/doma-spring-boot-sample-docker-compose/src/test/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+# Enable Docker Compose in tests
+spring.docker.compose.skip.in-tests=false
+spring.sql.init.mode=always
+spring.sql.init.schema-locations=classpath:schema-test.sql
```

Comment:
I don't like having to manage the data twice using schema-test.sql and schema.sql. Instead of using schema-test.sql for testing, perform the reset process in the test code.